src/components/Subscribe/Subscribe.tsx

```diff
@@ -51,6 +51,10 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
    * Subscribe success text
    */
   successText?: string;
+  /**
+   * Subscribe privacy text
+   */
+  privacyText?: string;
```

Comment:
are we going to have this privacy text in all areas where we use this component? We are making it conditional in the types but we are always rendering a p tag. We should conditionally render this element.